src/events/crawler/scrapers/USA/GA/index.js

```diff
@@ -180,7 +180,8 @@ const scraper = {
     const $trs = $table.find('tbody > tr');
     $trs.each((index, tr) => {
       const $tr = $(tr);
-      let county = geography.addCounty(parse.string($tr.find('td:first-child').text()));
+      const name = $tr.find('td:first-child').text();
+      let county = geography.addCounty(parse.string(name.replace('Dekalb', 'DeKalb')));
```

Comment:
Looks like a nice tactical fix for this.

Reply:
Personally, I feel the best way to fix it would be to ignore case for every county name because I have a suspension this isn't the only place capitalization will cause problems. But yeah, this works for now.

Reply:
Agreed. See this issue for more context:

https://github.com/lazd/coronadatascraper/issues/314